cscs-checks/microbenchmarks/simd/vc.py

```diff
@@ -0,0 +1,83 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class VcSimdTest(rfm.RegressionTest):
+    """
+    Testing https://github.com/VcDevel/Vc
+    Example performance report will look like:
+    > reframe --system dom:mc -p PrgEnv-gnu -r -c vc.py --performance-report
+        PERFORMANCE REPORT
+        -----------------------------------------------------------------------
+        VcSimdTest
+        - dom:mc
+           - PrgEnv-gnu
+              * speedup: 1.3813 cyc
+    > reframe --system dom:gpu -p PrgEnv-gnu -r -c vc.py --performance-report
+        PERFORMANCE REPORT
+        -----------------------------------------------------------------------
+        VcSimdTest
+        - dom:gpu
+           - PrgEnv-gnu
+              * speedup: 1.3824 cyc
+    """
+    def __init__(self):
+        self.valid_systems = ['*']
```

Comment:
Is it meant to be run on `login` as well?

Reply:
can be